ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,527 @@
+@prefix : <https://semopenalex.org/vocab#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix pmc: <https://www.ncbi.nlm.nih.gov/pmc/articles/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix fn: <http://www.w3.org/2005/xpath-functions#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix cito: <http://purl.org/spar/cito/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix prism: <http://prismstandard.org/namespaces/basic/2.0/> .
+@prefix rdf4j: <http://rdf4j.org/schema/rdf4j#> .
+@prefix wgs: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix gn: <http://www.geonames.org/ontology#> .
+@prefix fabio: <http://purl.org/spar/fabio/> .
+@prefix Default: <http://www.metaphacts.com/resource/> .
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix dbp: <http://dbpedia.org/property/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix sesame: <http://www.openrdf.org/schema/sesame#> .
+@prefix pm: <https://pubmed.ncbi.nlm.nih.gov/> .
+
+<https://semopenalex.org/vocab#> a owl:Ontology;
+  rdfs:label "SemOpenAlex Ontology"@en;
+  rdfs:comment "The Semantic OpenAlex Ontology, described using W3C RDF Schema and the Web Ontology Language OWL."@en .
+
+<https://semopenalex.org/class/Author> a owl:Class;
+  rdfs:label "Author"@en .
+
+<https://semopenalex.org/class/Institution> a owl:Class;
+  rdfs:label "Institution"@en .
+
+<https://semopenalex.org/property/hasRelatedWork> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/Work> .
+
+<https://semopenalex.org/class/Work> a owl:Class;
+  owl:equivalentClass fabio:Work;
+  rdfs:label "Work"@en .
+
+<https://semopenalex.org/property/hasAssociatedInstitution> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range <https://semopenalex.org/class/Institution> .
+
+<https://semopenalex.org/property/hasConcept> a owl:ObjectProperty;
+  rdfs:domain _:node61;
+  rdfs:range skos:Concept .
+
+<https://semopenalex.org/class/ConceptScore> a owl:Class;
+  rdfs:label "ConceptScore"@en .
+
+<https://semopenalex.org/property/hasConceptScore> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/ConceptScore> .
+
+cito:cites a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/Work> .
+
+<https://semopenalex.org/property/countsByYear> a owl:ObjectProperty;
+  rdfs:domain _:node64;
```

Comment:
how did you produce this ttl file? Could you try to apply pretty printing and inline bnodes?  I guess metaphactory ontology download should do this